plane/source/docs/apms-failsafe-function.rst

```diff
@@ -57,19 +57,19 @@ Meaning that when flying, our throttle values will range between 1100 -
 
 -  If we lose RC communication, the receiver if set up properly, will
    drop to the lowest known throttle value of ~900. This value falls
-   bellow the THR_FS_VALUE and will trigger the autopilot to go into
+   bellow the :ref:`THR_FS_VALUE<THR_FS_VALUE>` and will trigger the autopilot to go into
    failsafe mode.
--  First the autopilot will go into short failsafe (FS_SHORT_ACTN,
-   0=Disabled, 1=Enabled) when it detects loss of signal for more than
-   FS_SHORT_TIMEOUT sec. The default setting for short failsafe is Circle mode.
+-  First the autopilot will go into short failsafe (:ref:`FS_SHORT_ACTN<FS_SHORT_ACTN>` ),
+   when it detects loss of signal for more than :ref:`FS_SHORT_TIMEOUT<FS_SHORT_TIMEOUT>` sec. The default setting for short failsafe is Circle mode.
 -  If the RC signal is regained during the short failsafe, the flight
-   will return to auto mode.
--  If the loss of signal is longer than FS_LONG_TIMEOUT sec the autopilot will go
-   into long failsafe (FS_LONG_ACTN, 0=Disabled, 1=Enabled).
+   will return to the previous mode.
+-  If the loss of signal is longer than :ref:`FS_LONG_TIMEOUT<FS_LONG_TIMEOUT>` sec the autopilot will go into long failsafe :ref:`FS_LONG_ACTN<FS_LONG_ACTN>` .
 -  The default setting for long failsafe is RTL (Return to Launch).
--  Once the long failsafe (RTL mode) has been entered at the conclusion
-   of the short failsafe the RTL mode will continue even if your RC
-   signal is reacquired.
+
+
+.. note:: Once the long failsafe has been entered at the conclusion
+   of the short failsafe the :ref:`FS_LONG_ACTN<FS_LONG_ACTN>`  mode will continue even if your RC
+   signal is reacquired. Once reacquired, the mode can only be exited via a mode change.
```

Comment:
Should note that any other failsafe triggering IE battery can also cause a mode change. A RC failsafe doesn't lock you out of any other failsafes.

Reply:
fixed